firmware/lib/Gantry/Axis.cpp

```diff
@@ -111,18 +129,22 @@ void isr_limit_switch_x()
     uint32_t interrupt_time = millis();
     // If interrupts come faster than 10ms, assume it's a bounce and ignore
     if (interrupt_time - last_interrupt_time > 100) {
-        NVIC_DisableIRQ(axis_x.irq_velocity);
-        NVIC_DisableIRQ(axis_x.irq_accel);
+        stop_axis(&axis_x);
         LimitSwitchStatus status = (LimitSwitchStatus)digitalRead(LIMIT_SW_HOME_PIN_X);
         // DEBUG_PRINT_VAL("X Home Limit switch has been hit ", status);
-        if (axis_x.homing && (status == RELEASED)) {
-            axis_x.encoder.current = 0;
-            axis_x.homing = 0;
-            DEBUG_PRINT_VAL("X axis has been homed - SUCCESS", 1);
+        if (axis_x.homing) {
+            if (status == RELEASED) {
+                axis_x.encoder.current = 0;
+                axis_x.homing = false;
+                DEBUG_PRINT_VAL("X axis has been homed - SUCCESS", 1);
+            }
+            else if (status == PRESSED) {
+                DEBUG_PRINT_VAL("HOMING IN POSITIVE DIR ", 0);
```

Comment:
What does the `1` and `0` mean in `DEBUG_PRINT_VAL`?

Reply:
No clue, but gonna go through and delete all of these later